scientific_library/tvb/simulator/monitors.py

```diff
@@ -374,6 +401,49 @@ def sample(self, step, state):
             return [time, avg_stock]
 
 
+class Coupling(RawVoi):
```

Comment:
I think Coupling is getting a bit overloaded in TVB, and it would be preferable to make the name more specific.  For instance, are you montioring the afferent or efferent coupling variables?

Reply:
I just renamed to AfferentCoupling. I am not sure that Afferent is clear enough. However, I can also rename to Afferent if you insist.

Reply:
Yes agreeed that afférent alone is insufficient.  How about PostSynapticTrace?

Reply:
To be honest, I don't think that switching to a terminology that doesn't include "Coupling" is an accurate description.
Many of our models are anyway too phenomenological for "postsynaptic" to really mean something.
Everywhere in TVB we call "Coupling", actually node_coupling, or local_coupling this quantity, and therefore, the Monitor is better to use the same wording.
I agreed though with the Afferent distinction because it has to be made against EfferentCoupling.

Reply:
The pre/post synaptic distinction is held if loosely by whether something occurs before or after the summation, but if we can agree to use AfferentCoupling that's fine too.